src/GRA.Controllers/ViewModel/MissionControl/Challenges/ChallengesDetailViewModel.cs

```diff
@@ -24,6 +24,8 @@ public class ChallengesDetailViewModel
         public string BadgeMakerUrl { get; set; }
         public bool UseBadgeMaker { get; set; }
         public string BadgeMakerImage { get; set; }
+        [DisplayName("Badge Alt-Text")]
+        public string BadgeAltText { get; set; }
```

Comment:
Property should have the MaxLength attribute to help check for ModelErrors.

Reply:
Fixed in commit d65c7ad1f42bf397ccad695a33bd75da81206656